src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackService.java

```diff
@@ -66,36 +63,49 @@ public CreateCaseCallbackService(
      * @return Either list of errors or map of changes - new case reference
      */
     public ProcessResult process(CcdCallbackRequest request, String idamToken, String userId) {
-        ProcessResult result = assertAllowToAccess(request.getCaseDetails(), request.getEventId())
-            .flatMap(theVoid -> validator
-                .getValidation(request.getCaseDetails())
-                .combine(getServiceConfig(request.getCaseDetails()).mapError(Array::of))
-                .ap((exceptionRecord, configItem) -> createNewCase(
-                    exceptionRecord,
-                    configItem,
-                    request.getCaseDetails().getId(),
-                    request.isIgnoreWarnings(),
-                    idamToken,
-                    userId
-                ))
-                .mapError(errors -> errors.flatMap(Function.identity()))
-                .flatMap(Function.identity())
-                .mapError(Seq::asJava)
-            )
+        Validation<String, Void> canAccess = assertAllowToAccess(request.getCaseDetails(), request.getEventId());
+
+        if (canAccess.isInvalid()) {
+            // log happens in assertion method
+            return new ProcessResult(emptyList(), singletonList(canAccess.getError()));
+        }
```

Comment:
I like this change. As for what is returned, this case should be a 500 result, so throwing an exception would be ideal. Not necessarily for this PR.

Reply:
Here or somewhere else, we could log these errors, as they come from us.